### PR TITLE
Modify the selefra category

### PR DESCRIPTION
**Pull request by FelixsJiang**
## Requirements

 * [x] This is Software as a Service not self hosted
 * [x] It has a free tier not just a free trial
 * [x] The submission mentions what is free
 * [x] The submission is not already present in the list
 * [x] The service has contact details of those running it and a privacy policy

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -157,6 +157,7 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
 ## Cloud management solutions
   * [Brainboard](https://www.brainboard.co) - Collaborative solution to visually build and manage cloud infrastructures from end-to-end.
   * [Pulumi](https://www.pulumi.com/) — Modern infrastructure as code platform that allows you to use familiar programming languages and tools to build, deploy, and manage cloud infrastructure.
+  * [Selefra](https://www.selefra.io/) - An open-source policy-as-code software that provides analytics for multi-cloud and SaaS.
```
Comment (ripienaar) [inline README.md:160]: ok since you're modifying it please state the features of the free tier like other entries do

Comment (FelixsJiang) [reply README.md:160]: [Selefra](https://selefra.io/pricing) This is our SaaS pricing website. Our free version offers 1 project, 1 connection, and 1 member for users to use. New users are given a 7-day trial of the team version. However, our free version also allows full access to all functionalities, with only limited quantities.

Comment (ripienaar) [reply README.md:160]: Snyk example: ```Can find and fix known security vulnerabilities in your open source dependencies. Unlimited tests and remediation for open source projects. Limited to 200 tests/month for your private projects.``` I just need you to add a short 1 sentence about whats free

Comment (FelixsJiang) [reply README.md]: I modified the project introduction statement as Selefra free tier offers individual users to scan one single-source connection to discover security and compliance issues on the configurations level of cloud/infra/SaaS.

**Pull request merged**